src/snowflake/snowpark/_internal/open_telemetry.py

```diff
@@ -56,6 +65,60 @@ def open_telemetry_context_manager(func, dataframe):
         yield
 
 
+@contextmanager
+def open_telemetry_udf_context_manager(func, parameters):
+    # trace when required package is installed
+    if open_telemetry_found:
+        class_name = func.__qualname__
+        name = func.__name__
+        tracer = trace.get_tracer(f"snow.snowpark.stored_procedure:{class_name}")
```

Comment:
Why `snow.snowpark.stored_procedure`? This is the scope name?
I thought we used just the class_name (DataFrame) for the action APIs?

Reply:
Its a typo, I'll change it